runtime/src/accounts_db.rs

```diff
@@ -1331,9 +1331,8 @@ impl AccountsDB {
     }
 
     pub fn include_owner_in_hash(slot: Slot) -> bool {
-        // Account hashing updated to include owner activates at this slot on the testnet
-        // DANGER: Remove this guard ABSOLUTELY before the mainnet-beta transitions to v1.1.
-        slot >= 14_000_000
+        // Account hashing updated to include owner activates at this slot on the testnet/mainnet-beta
+        slot >= 11_000_000
```

Comment:
@mvines Hhm, odd is this really true? Sorry but this should be 12_500_000: https://github.com/solana-labs/solana/blob/v1.0/runtime/src/accounts_db.rs#L1070

I'm testing now. :)

Reply:
Whoa.  That is so strange, I found that value from checking one of my local git clones I think.  I wonder if I accidentally found an older version of the code that was never merged.

`12_500_000` certainly looks to be correct.

Reply:
I guess it doesn't matter, a lower value here shouldn't be harmful since mainnet-beta is in the 13m slot range now

Reply:
Yeah, that might be true. :) Though, for that, all of live accounts must be updated after 12_500_000. Or, no account is updated between 11_000_000 and 12_500_000. Or I'm missing some. :)

I'm testing this now and shortly after create a follow-up pr if needed!

Reply:
@mvines FYI: I ended up creating a follow up pr: https://github.com/solana-labs/solana/pull/10243 :)